builtin/docker/plugin.proto

```diff
@@ -22,16 +23,16 @@ message Image {
     // registry is set if the image is in a remote registry. This value
     // might mean the image is local, too, but we never formally "pulled"
     // it so we aren't sure. The image should be treated as remote.
-    RegistryLocation registry = 3;
+    RegistryLocation registry = 4;
 
     // docker is set if the image is in a local Docker daemon registry.
-    google.protobuf.Empty docker = 4;
+    google.protobuf.Empty docker = 5;
 
     // Waypoint 0.6 and earlier supported "img" as a Dockerless builder.
     // We removed this in 0.7, hence we prefix this now with "unused". We
     // keep the field around so we can detect old clients sending it and
     // show an error. Do not set this.
-    google.protobuf.Empty unused_img = 5;
+    google.protobuf.Empty unused_img = 6;
```

Comment:
if I remember right, these numbers should _never_ be changed. Otherwise old proto clients will use the wrong variables during communication. It's totally fine to make `architecture` the _next_ number here, no need to bump the other ones.

Reply:
Ah TIL! and that makes sense